killerbee/zbwardrive/findgps.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Helpful script to find where the gps is
+
+for sysdevpath in $(find /sys/bus/usb/devices/usb*/ -name dev); do
+    (
+        syspath="${sysdevpath%/dev}"
+        devname="$(udevadm info -q name -p $syspath)"
+        eval "$(udevadm info -q property --export -p $syspath)"
+	if [[ "$ID_SERIAL" = *"Prolific"* ]]
```

Comment:
What about for other vendors? Maybe add in the comments that this is only tested so far with device ___ version ___ and people are encouraged to add support for other devices, etc?